### ISSUE TITLE
Extra space on Navbar

### ISSUE THREAD

**Issue by andregoncalvesdev** [react-native-fellowship/react-native-navbar#174]
I added my Navbar component like this:
`<NavigationBar
          style={{backgroundColor: "#f1f1f1"}}
          leftButton={ <BackButton onPress={() => this.props.navigator.pop()}/> } />`

The View container dont have paddings or margins and the status bar neither. 

![screen shot 2016-06-06 at 5 29 30 pm](https://cloud.githubusercontent.com/assets/9271177/15829630/6cd44a90-2c0c-11e6-8aed-d4de5ef8bf7a.png)

Thanks in advance

**Comment by kikacaty** [comment]
Yes I am having the same issue for that with "react-native-navbar": "~1.1.6" and "react-native": "0.21.0".

**Comment by vitalyrotari** [comment]
same on `react-native@0.32.0` and `react-native-navbar@1.5.0`

**Comment by UXDart** [comment]
any news on this bug?
